spec/spec_helper.rb

```diff
@@ -1,10 +1,11 @@
 require 'simplecov'
-require 'undercover'
 
 SimpleCov.start do
-  add_filter(%r{\/spec\/})
+  add_filter(%r{/spec/})
 end
 
+require_relative '../config/autoload'
+# require_relative './constants_for_spec'
```

Comment:
Hm, you don't use constants_for_spec.rb, but don't remove file. Why you didn't remove it?
Also, remove this line

Reply:
Removed